mini-accumulo-cluster/pom.xml

```diff
@@ -27,7 +27,7 @@
 
     <properties>
         <!-- Using older version as there were problems running the 1.7.2 mini accumulo cluster -->
-        <accumulo.version>1.6.5</accumulo.version>
+        <accumulo.version>1.8.1</accumulo.version>
```

Comment:
This entire properties block could actually now be removed, as the accumulo.version property is set to 1.8.1 in a parent pom.